pom.xml

```diff
@@ -80,8 +80,8 @@
         <configuration>
           <instructions>
             <Export-Package>
-              org.everit.osgi.ecm.component,
-              org.everit.osgi.ecm.component.resource
+              org.everit.osgi.ecm.component;version="1.1.0",
```

Comment:
I see only javadoc change that is only a bugfix change. You sure that it is necessary to increment the minor version?